### PR TITLE
✅ Build Script: Write a script that build & deploy the package

### PR DESCRIPTION
**Pull request by yashksaini-coder**
This PR after further testing, has finally fixed the issue #167 to automate the **Build & Deploy** script.

IT uses the previous approach we discussed in #167, 

[sneaky-package-167.webm](https://github.com/user-attachments/assets/94161d30-32d9-42e7-8db1-617e819c1b52)

---

@this-is-yaash Review this PR and the code changes and kindly assign labels to this PR.

This pull request includes significant changes to the build process and documentation of the project. The most important changes are the addition of a new build script, removal of contribution guidelines, and updates to the `setup.py` file.

### Build Process Improvements:
* [`build.py`](diffhunk://#diff-b70bfb027a8ef3ec4ab07a53a3b8ba83e63334ba3ce5ca6d9d773900ed452e5eR1-R57): Introduced a new build script to dynamically create `setup.py`, build the package, and upload it to PyPI using environment variables.

### Documentation Changes:
* [`docs/Contributing.md`](diffhunk://#diff-0474e525df6c10faef7f4f0dbee275b9f58d1312320b5a42d04be66850ec32a7L1-L116): Removed the detailed contribution guidelines, which included instructions for setting up the development environment, understanding the codebase, and submitting contributions.
* [`docs/contributing.md`](diffhunk://#diff-419d79ae7d440d49b3fa5fa7739f5f3a359e3ca42c3fa942e5604ae3474eb35bL1-L109): Removed the detailed contribution guidelines, similar to the changes in `docs/Contributing.md`.

### Package Configuration:
* [`setup.py`](diffhunk://#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7L1-R10): Updated package metadata, including the package name, version, description, author name, and author email.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -159,5 +159,5 @@ cython_debug/
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
 
-# Custom directory
+# Custom directory & files
 matplotlib_visual/code_snippets
```
Comment (this-is-yaash) [inline .gitignore:163]:
```suggestion
matplotlib_visual/stash
```

**Comment by this-is-yaash** [comment]
@yashksaini-coder I think we don't have to enter the token when prompting.
Make it fetch from the `.env`.

Then we'll merge it!

File: docs/Contributing.md
Comment (this-is-yaash) [inline docs/Contributing.md:1]: don't delete this file!
!1

File: docs/contributing.md
Comment (this-is-yaash) [inline docs/contributing.md:1]: don't delete this file!
!2

File: setup.py
Comment (this-is-yaash) [inline setup.py:1]: revert back to original changes

**Comment by yashksaini-coder** [comment]
We can discuss the progress and work done so far, and resolving the Contributing.md file issue tomorrow, for now closing this PR

**Pull request closed without merging**